repo/packages/W/weavescope-probe/2/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "packagingVersion": "3.0",
+  "name": "weavescope-probe",
+  "version": "1.2.1",
```

Comment:
It's just that we were debating if we could avoid having to open a PR for each version bump...